docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Keyacrd? (have no idea how to call it)
+version: 0.0.0
+status: Draft
+authors: Roman Volosovskyi <roman@status.im>
+redirect_from:
+  - /keycard.html
+---
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Multiaccount creation](#Multiaccount-creationrestoring)
+4. [Multiaccount restoring via pairing](#Multiaccount-restoring-via-pairing)
+5. [Multiaccount unlocking](#Multiaccount-unlocking)
+6. [Where are the keys stored?](#Where-are-the-keys-stored)
+
+## Abstract
+
+In this specification, we describe how Satus communicates with Keycard to create, store and use multiaccount.
+
+## Definitions
+
+
+| Term | Description |
```

Comment:
Multiaccount
Keycard Hardwallet

Possibly things like pin, mnemonic etc, not sure.

Refer to ACCOUNT spec and any existing keycard specs (?) as well

Reply:
`Multiaccount, mnemonic` - shouldn't it be a part of a separate doc? It is not keycard specific term, I would rather add a reference (if one exists)